packages/airnode-deployer/src/utils/cli.ts

```diff
@@ -1,3 +1,6 @@
+import * as crypto from 'crypto';
+import { format } from 'date-fns-tz';
```

Comment:
Do we need a new library for this? Why not just use the format from date-fns ?

Reply:
Not sure if that is worth introducing a new library. But I don't have a strong preference.

Reply:
So, I've checked and there is a difference based on the user's locale settings (e.g. difference between GMT+2 and CEST). I think it's worth keeping it to improve the UX but not a big deal though :man_shrugging: